guillotina/schema/utils.py

```diff
@@ -31,4 +31,6 @@ def get_default_from_schema(context, schema, fieldname, default=None):
             return deepcopy(field.defaultFactory())
     if field.default is not None:
         return deepcopy(field.default)
+    if field.default != default:
```

Comment:
If the field has `default = None` and we ask for another value we return `None`. Which use case covers this line?

Reply:
Right here: https://github.com/plone/guillotina/blob/master/guillotina/content.py#L252